templates/base.html

```diff
@@ -7,8 +7,11 @@
   <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='/css/background.css') }}" >
   <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='/css/button.css') }}" >
   <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='/css/popup_frame.css') }}" >
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='/css/lightning.css') }}" >
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/animate.min.css')}}">
```

Comment:
If these file are optional need, include them in `{% block head %}` in specific page.

Reply:
done